src/utils/rules.ts

```diff
@@ -13,13 +13,14 @@ const propThemeMap = {
   bg: 'bg',
 };
 
+// TODO FIX
 export const styleColor = ({ prop, cssProperty }: { prop: string; cssProperty?: string }) => {
   const resolvedProp = cssProperty || prop;
 
   // create a cache to check and break circular references
   // @ts-ignore FIXME
   const fn = (props: any, cache: any[] = []) => {
-    let val = props[prop];
+    let val = props[prop] || '';
```

Comment:
There is no need in a `''`, since if `val` is falsy function will always return a `null`.

```suggestion
    let val = props[prop];
```

Reply:
Updated!